test/e2e-nokey/rag.e2e.test.js

```diff
@@ -2,8 +2,11 @@ const { test, expect } = require('@playwright/test');
 const fs = require('fs');
 const path = require('path');
 
-test.describe('RAG File Upload Integration', () => {
+test.describe('RAG Integration End-to-End Tests', () => {
   test.describe.configure({ mode: 'serial' });
+
+  let sharedPage;
+
   // Helper to remove 'test-*' files from RAG input and 'ingested' dirs
   const cleanupTestFiles = () => {
     const ragInputDir = path.join(__dirname, '../../rag_input');
```

Comment:
We don't ship the project with any files in that path.  I think this is going to fail in CI.